### PR TITLE
Add 1.28 release interview blog post

### PR DESCRIPTION
**Pull request by craigbox**
Publish an abridged transcript of my podcast interview with Grace Nguyen as a kubernetes.io blog post.

See also: #36051, #33280, #30688, #29146, #22882, #22716, #17963 et al.

### CODE REVIEW

**Comment by craigbox** [comment]
Much better. 
https://deploy-preview-43599--kubernetes-io-main-staging.netlify.app/blog/2023/10/20/plants-process-and-parties-the-kubernetes-1.28-release-interview/

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:150]:
```suggestion
```

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:159]:
```suggestion
Right; this one is close to my heart. It is a SIG Release KEP, [publishing on community infrastructure](https://github.com/kubernetes/enhancements/issues/1731). I'm not sure if you know, but as part of my branch management associate role in 1.27, I had the opportunity to cut a few releases. It takes up to 12 hours sometimes. And now, we are hoping that the process only includes release managers, so we don't have to call up the folks at Google and, you know, that won't lengthen the process anymore.
```

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:162]:
```suggestion
There's so much work in place to bring that down. I think 12 hours is on the shorter end of it. Unfortunately, we have faced a situation where we have to, you know, switch the release manager because it's just so late at night for them. 
```

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:206]:
```suggestion
**As we've alluded to a little bit throughout our conversation, there are a lot of people in the Kubernetes community who want a better term, and had "a lot of experience" at running these systems. Then there are, of course, a lot of people who are just at the beginning of their careers; like yourself, at university. How do you see the difference between how those groups interact? Is there one team throughout, or what do you think that each can learn from the other?**<br>
```

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:209]:
```suggestion
**I know the CNCF goes to a lot of effort to enable new people to take part. Is there anything that you can say about how people can get involved?**<br>
```

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:210]:
```suggestion
Firstly, I think SIG Release has started a wonderful tradition, or system, of helping new folks [join the release team as a shadow](https://github.com/kubernetes/sig-release/blob/master/release-team/shadows.md), and helping them grow into bigger positions, like leads. I think other SIGs are also following that template as well. But a big part of me joining and sticking with the community has been the ability to go to conferences. As I said, my first conference was KubeCon, when I was not involved in the community at all. And so a big shout-out to the CNCF and the companies that sponsor the Dan Kohn and the speaker scholarships. They have been the sole reason that I was able to attend KubeCon, meet people, and feel the power of the community. 
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
1. Wrap text of different paragraphs written in a single line to smaller lines of optimal length.
2. While reading this blog, at some places it became difficult to understand who said that word or line. Add, name of the person that is speaking at starting of each line/para. For example -
Person 1: Hi! My name is XYZ
Person 2: Hello, Nice to meet you
3. Here are some other suggestions below:-

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:68]: suggestion is not grammatically correct, and isn't reflective of what was spoken

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:71]: It could have been spoken in a better way too 🤣

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:110]: suggestion is not grammatically correct, and isn't reflective of what was spoken

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:123]: This is what was said, and it bears repeating for emphasis.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:125]: The correct thing to link is the blog post including the words "blog post", and adding quotes would make no sense here.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:150]: This is what was actually spoken.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:159]: suggestion is not grammatically correct, and isn't reflective of what was spoken.

(Will take s/that/the/ though, thank you!)

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:162]: This isn't reflective of how Grace speaks, and is too large a change for a verbatim transcript.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:206]: suggestion is not grammatically correct, and isn't reflective of what was spoken

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:210]: I think this is an acceptable link because the page does refer to why the process exists and how it helps ("The Shadow system is an apprenticeship model, like those used by union tradespeople."), and is not just the method of joining.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:209]: stylistic choice only; will remain with what I spoke

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:63]: this would be OK as an addition, but throughout I've tried to keep Grace's tone, and not lose her flavour of speech.

File: content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md
Comment (craigbox) [reply content/en/blog/_posts/2023-10-20-kubernetes-1.28-release-interview.md:41]: [I did some research on grammatical rules for this one!](https://english.stackexchange.com/questions/435246/should-i-put-a-comma-after-is-if-its-followed-by-a-prepositional-phrase) The comma would probably more correctly go _before_ the "is", but I think that as-is reflects how it was spoken.

**Comment by craigbox** [comment]
Thank you for your review, @aj11anuj.  Please be aware of the context of this post: it's a (mostly) verbatim transcript of a podcast interview.  Minor corrections have been made for clarity and ease of reading, but throughout I've attempted to keep the flavour of the language differences between Grace and myself, as it makes up part of the story.  

> 1. Wrap text of different paragraphs written in a single line to smaller lines of optimal length.

This doesn't make sense for a transcript, and I don't think any of the paragraphs are long enough that it would add value to do so.

> 2. While reading this blog, at some places it became difficult to understand who said that word or line. Add, name of the person that is speaking at starting of each line/para. For example -
>    Person 1: Hi! My name is XYZ
>    Person 2: Hello, Nice to meet you

Please have a look at the last in this series: https://kubernetes.io/blog/2022/08/18/stargazing-solutions-and-staycations-the-kubernetes-1.24-release-interview/

Up to this point I have gone with a variation on the format provided by the transcription service, which uses GUEST NAME: at the start of the line.  I have bolded the lines I spoke and left the guest lines.

Grace gave much more succinct answers than James, and the usual method left a lot of lines that just came after each other. Given that, I found that the "CRAIG BOX" and "GRACE NGUYEN" that started each line were distracting. Instead, I adopted a more succinct format that I've often seen used in magazine articles in a subsequent commit.

I'm happy to go either way, but the choice was made thoughtfully.

**Review by nate-double-u — COMMENTED** [review]
> Instead, I adopted a more succinct format that I've often seen used in magazine articles ([example](https://www.vulture.com/article/conan-obrien-late-night-legacy-interview.html)) in a subsequent commit.

This is a good, clear format.

We'll need to update the date to sometime in the future--maybe Wednesday or Thursday of next week? 

Otherwise I agree, this is in line with previous release interview transcript blogs.

/lgtm

**Comment by nate-double-u** [comment]
/hold
for date change

**Comment by nate-double-u** [comment]
Reapplying lgtm

/lgtm

Folks can remove the hold when needed/approved.

**Comment by craigbox** [comment]
/assign @onlydole

Taylor, could you please take a look at this?  Nate's been through and LGTM, just need approval.

File: content/en/blog/_posts/2023-10-24-kubernetes-1.28-release-interview.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2023-10-24-kubernetes-1.28-release-interview.md:167]: The "Planternetes" link is currently leading to a 404 error ([blog preview.](https://deploy-preview-43599--kubernetes-io-main-staging.netlify.app/blog/2023/10/24/plants-process-and-parties-the-kubernetes-1.28-release-interview/)). We should update it to point to the correct location, which is `/images/blog/2023-08-15-kubernetes-1.28-blog/kubernetes-1.28.png`
However, instead of directly linking to the image asset, we could also consider linking to the release blog section, where the release theme logo is displayed (refer to the suggestion below).
```suggestion
**The theme for this release is "[Planternetes](/blog/2023/08/15/kubernetes-v1-28-release/#release-theme-and-logo)". That's going to need some explanation, I feel.**<br>
```

Comment (craigbox) [reply content/en/blog/_posts/2023-10-24-kubernetes-1.28-release-interview.md:167]: Thanks!  I accidentally lost an `images/` when I changed to a relative URL. Thats a good suggestion too.

**Review by onlydole — APPROVED** [review]
Thank you for sharing, @craigbox

**Review by dipesh-rawat — COMMENTED** [review]
Thanks!
/lgtm

[Blog Preview](https://deploy-preview-43599--kubernetes-io-main-staging.netlify.app/blog/2023/10/24/plants-process-and-parties-the-kubernetes-1.28-release-interview/)

**Pull request merged**
